modules/ROOT/nav.adoc

```diff
@@ -398,6 +398,16 @@
 ** xref:tinymce-and-cors.adoc[Cross-Origin Resource Sharing (CORS)]
 * Release information
 ** xref:release-notes.adoc[Release notes for TinyMCE 6]
+// *** TinyMCE 6.4.2
```

Comment:
I assume these are supposed to be commented out

Reply:
Yes, as they are just a placeholder for now.